easybuild/framework/easyblock.py

```diff
@@ -811,8 +828,8 @@ def obtain_file(self, filename, extension=False, urls=None, download_filename=No
             srcpaths = source_paths()
 
         # We don't account for the checksums file in the progress bar
-        if filename != 'checksum.json':
-            update_progress_bar(PROGRESS_BAR_DOWNLOAD_ALL, label=filename)
+        if filename != 'checksums.json':
```

Comment:
This typo fix is a clear indication that we should have a constant for this, fixed in https://github.com/smoors/easybuild-framework/pull/2